src/content/en/2021/compression.md

```diff
@@ -184,20 +191,24 @@ Gzip compression is applied largely around compression level 6, extending to lev
+  description="Stacked bar charts showing the usage of compression levels of responses using Gzip, for compression level 1 to 9, as well as `opt` which corresponds to optimizers such as Zopfli. It shows peaks at level 1 and at level 6, as well as a smaller peak at level 9. Each level is also broken down by content type.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQtfyTM9VEweN_Hli3IuxxqU1CRap4V5Q28baEs7aEBResoPRgk9Dwp1m_vdS9lzNlfO8J4hZN7GPT7/pubchart?oid=586666706&format=interactive",
   sheets_gid="150560131"
   )
 }}
 
-## How to analyze compression on your sites
+{# TODO - should data be broken down more to say which content type? #}
+
+## How to analyze compression on sites
 
-You can check which content of your website is already using HTTP compression in the [Firefox Developer Tools](https://developer.mozilla.org/en-US/docs/Tools) or the <a hreflang="en" href="https://developers.google.com/web/tools/chrome-devtools">Chrome DevTools</a>. In the developer tools, open the Network tab and reload your site. A list of responses such as HTML, CSS, JavaScript, fonts and images should appear. To see which ones are compressed, you can check the content encoding in their response header. You can enable a column to easily see this for all responses at once. To do this, right click on the column titles, and in the menu navigate to Response Headers and enable Content-Encoding.
+To check which content of a website is already using HTTP compression, the [Firefox Developer Tools](https://developer.mozilla.org/en-US/docs/Tools) or the <a hreflang="en" href="https://developers.google.com/web/tools/chrome-devtools">Chrome DevTools</a> can be used. In the developer tools, open the Network tab and reload your site. A list of responses such as HTML, CSS, JavaScript, fonts and images should appear. To see which ones are compressed, you can check the content encoding in their response header. You can enable a column to easily see this for all responses at once. To do this, right click on the column titles, and in the menu navigate to Response Headers and enable Content-Encoding.
 
 Responses that are Gzip compressed will show "gzip", while those compressed with Brotli will show "br". If the value is blank, no HTTP compression is used. For images this is normal, since these resources are already compressed on their own.
 
 If you hover the mouse over the values in the Size column, you can also see "transferred over network" and "resource size" to compare the compressed and actual sizes. This data can also be seen for the entire site: this is indicated as "size" / "transferred size" in Firefox and "transferred" and "resources" in Chrome on the bottom left hand side of the Network tab.
 
+{# TODO is this section above necessary? it has less to do with discussing data #}
```

Comment:
It makes some sense, in that it tells how to analyze your own data (or other websites). If this gets a bit too detailed, it could be shortened by removing the last paragraph above this comment about the Size column and the transfered network size (since this is just one of the many things you can do). The part higher up about the response header is useful in general though as it shows the HTTP content-encoding this chapter is talking about in use. What do you think?

Reply:
I removed the part about the sizes, and changed the first paragraph to say analysis of "a" site rather than "your" site because this is generally applicable, a way to analyze compression usage anywhere, to make this part slightly less instructional. The screenshot remains a useful addition imho thanks to showing the content encodings visually